app/views/editor/posts/_form.html.haml

```diff
@@ -22,15 +22,17 @@
 
 :javascript
   $(function() {
-    function applySelect2($element) {
+    function applySelect2($element, allowClear) {
       $element.select2({
-        placeholder: '選択してください'
+        placeholder: '選択してください',
+        allowClear: allowClear
       });
     }
 
-    applySelect2($('select'));
+    applySelect2($('select#post_category_id'), false);
```

Comment:
I've fixed.
(In previous version, I don't know how to merge JavaScript object like Ruby's)